src/main/ml-modules/root/com.marklogic.smart-mastering/matcher.xqy

```diff
@@ -51,16 +56,18 @@ as element(results)
  : @param $document  document to find matches for
  : @param $options-name  name previously associated with match options using matcher:save-options
  : @param $include-matches  whether the response should list the matched properties for each potential match
+ : @param $filter-query  a cts:query used to filter the matched results
```

Comment:
"a cts:query used to restrict matches to a set, such as a specific entity type or collection"